google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerDialect.java

```diff
@@ -598,16 +818,34 @@ public void lock(Serializable id, Object version, Object object, int timeout,
   static class EmptyExporter<T extends Exportable> implements Exporter<T> {
 
     @Override
-    public String[] getSqlCreateStrings(T exportable, Metadata metadata) {
+    public String[] getSqlCreateStrings(
+        T exportable, JdbcServices jdbcServices) {
```

Comment:
nit: Why arguments on a new line?
Seems like unnecessary diff fuzz. Also, a bunch more instances below.

Reply:
Yeah this was done by the intellij auto formatting. I'll look into how to import Hibernate checkstyle for us; that would be ideal moving forward.